src/main/webapp/courseLoader.js

```diff
@@ -8,13 +8,23 @@
   const termKey = urlParams.get("term-key");
   const courseKey = urlParams.get("course-key");
 
+  function setProfessorUrl(profName, profKey) {
+    const profLink = document.getElementById("prof-link");
+    const url = new URL("/professor.html", window.location.origin);
+    url.searchParams.set("prof-key", profKey);
+    url.searchParams.set("prof-name", profName);
+    profLink.setAttribute("href", url);
+  }
+
   function fillTitles() {
     fetch(`/term-info?term-key=${termKey}&course-key=${courseKey}`)
       .then((response) => response.json())
       .then((termInfo) => {
         document.getElementById("course-name").innerHTML = termInfo[0];
```

Comment:
Ideally these were all named properties rather than indexes. It would be much more clear (and harder to screw up) if this was instead `document.getElementById("term-name").innerHTML = termInfo.name;`.

Reply:
Will do in a separate PR that refactors some stuff